GitSearch/Models/Constants.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Constants.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/21/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import UIKit
+
+enum Constants {
```

Comment:
Where these constants are used?

Reply:
Later, when I'll merge this thin into master